browser/ui/views/split_view/split_view_browsertest.cc

```diff
@@ -990,448 +925,3 @@ IN_PROC_BROWSER_TEST_P(SplitViewCommonBrowserTest, SplitViewCloseTabTest) {
-  brave::NewSplitViewForTab(browser());
-
-  // In split view tile, both contents are visible and have its border.
-  EXPECT_TRUE(browser_view().contents_container()->GetVisible());
-  EXPECT_TRUE(browser_view().contents_container()->GetBorder());
-  EXPECT_TRUE(secondary_contents_container().GetVisible());
-  EXPECT_TRUE(secondary_contents_container().GetBorder());
-
-  // Simulate tab-fullscreen state change.
-  FullscreenController* fullscreen_controller = browser()
-                                                    ->GetFeatures()
-                                                    .exclusive_access_manager()
-                                                    ->fullscreen_controller();
-  fullscreen_controller->set_is_tab_fullscreen_for_testing(true);
-  split_view().OnFullscreenStateChanged();
-
-  // In tab full screen, only primary content is visible w/o border.
-  EXPECT_TRUE(browser_view().contents_container()->GetVisible());
-  EXPECT_FALSE(browser_view().contents_container()->GetBorder());
-  EXPECT_FALSE(secondary_contents_container().GetVisible());
-  EXPECT_FALSE(secondary_contents_container().GetBorder());
-
-  fullscreen_controller->set_is_tab_fullscreen_for_testing(false);
-  split_view().OnFullscreenStateChanged();
-
-  EXPECT_TRUE(browser_view().contents_container()->GetVisible());
-  EXPECT_TRUE(browser_view().contents_container()->GetBorder());
-  EXPECT_TRUE(secondary_contents_container().GetVisible());
-  EXPECT_TRUE(secondary_contents_container().GetBorder());
-}
-
-class SplitViewCookieBrowserTest : public SplitViewBrowserTest {
- public:
-  void SetUpOnMainThread() override {
-    SplitViewBrowserTest::SetUpOnMainThread();
-    embedded_test_server()->RegisterRequestHandler(base::BindRepeating(
-        [](const net::test_server::HttpRequest& request)
-            -> std::unique_ptr<net::test_server::HttpResponse> {
-          auto response =
-              std::make_unique<net::test_server::BasicHttpResponse>();
-          response->set_content(request.headers.count("Cookie")
-                                    ? request.headers.at("Cookie")
-                                    : "NONE");
-          response->set_code(net::HTTP_OK);
-          return response;
-        }));
-
-    ASSERT_TRUE(embedded_test_server()->Start());
-  }
-
-  void TearDownOnMainThread() override {
-    ASSERT_TRUE(embedded_test_server()->ShutdownAndWaitUntilComplete());
-    SplitViewBrowserTest::TearDownOnMainThread();
-  }
-};
-
-class MockBraveRenderViewContextMenu : public BraveRenderViewContextMenu {
- public:
-  MockBraveRenderViewContextMenu(content::RenderFrameHost& render_frame_host,
-                                 const content::ContextMenuParams& params)
-      : BraveRenderViewContextMenu(render_frame_host, params) {}
-  ~MockBraveRenderViewContextMenu() override = default;
-
-  void Show() override {}
-};
-
-IN_PROC_BROWSER_TEST_F(SplitViewCookieBrowserTest, CookieTest) {
```

Comment:
Deleted test code for cookie sharing issue in BraveSplitViewtest.
It fixed for BraveSplitView.
Original PR: https://github.com/brave/brave-core/pull/30390

Chromium also has fix for same issue in SideBySide.
https://chromium-review.googlesource.com/c/chromium/src/+/6838198